### PR TITLE
Feature/support object type

### PR DESCRIPTION
**Pull request by pnadolny13**
- support object type columns
- deserialize json string when its received by the tap and is of type object

### CODE REVIEW

File: tap_snowflake/__init__.py [hunk 1]
```diff
@@ -97,6 +97,9 @@ def schema_for_column(c):
         result.type = ['null', 'string']
         result.format = 'binary'
 
+    elif data_type == 'object':
```
Comment (cwatkins989) [inline tap_snowflake/__init__.py:100]: I'm wondering if this should use a mapping for `SEMI_STRUCTURED`. _Object_ is one of three types: https://docs.snowflake.com/en/sql-reference/data-types-semistructured.html.

I think they can all be handled very similarly here.

**Review by cwatkins989 — CHANGES_REQUESTED** [review]
Think we can extend coverage here for all Semi-Structured types pretty easily. Also, is this supposed to point to `master`, intentionally?

File: tap_snowflake/__init__.py (hunk 1 above)
Comment (pnadolny13) [reply tap_snowflake/__init__.py:100]: I know that it works for object type but I dont think this workflow generally handles variant properly. Variants can hold parquet and XML too which would throw an exception when we try to json.loads(xml_data)

**Comment by pnadolny13** [comment]
> Think we can extend coverage here for all Semi-Structured types pretty easily. Also, is this supposed to point to `master`, intentionally?

I did intentionally point this to master since its a single change that well release immediately, didnt feel like the develop/master flow was necessary...although I'm about to make the look back window changes so ill release it all together then

File: tap_snowflake/__init__.py (hunk 1 above)
Comment (cwatkins989) [reply tap_snowflake/__init__.py:100]: fair enough

**Review by cwatkins989 — APPROVED** [review]

**Pull request merged**